raspbian/applications/vcgencmd.md

```diff
@@ -0,0 +1,131 @@
+## VCGENCMD
+
+`vcgencmd` is a command line utility that can get various pieces of information from the Videocore4 GPU on the Raspberry Pi. Much of the information available is only of use to internal Raspberry Pi developers, but there are number of very useful options available to end users that will be described here.
+
+The source for the application can be found on our github page [here](https://github.com/raspberrypi/userland/tree/master/host_applications/linux/apps/gencmd).
+
+
+### Usage
+
+To get a list of all the commands that `vcgencmd` supports, type `vcgencmd commands`.
+
+Some of the more useful  commands are described below.
+
+### Commands 
+
+#### vcos
+
+The `vcos` cammand has a number of sub commands
+
+`version` Displays the build date and version of the firmware on the VideoCore.
+`log status` Displays the error log status of the various VideoCore software areas.
+
+#### version
+
+Displays the build date and version of the firmware on the VideoCore.
+
+#### get_camera
+
+Displays the enabled  and detected state of the official camera. 1 means yes, 0 means no. Whilst all firmware will support the camera, this support needs to be enabled by using [raspi-config](../../configuration/raspi-config.md).
+
+#### get_throttled
+
+Returns the throttled state of the system. This is a bit pattern.
+
+| Bit | Meaning |
+|:---:|---------|
+| 0   | Under-voltage detected |
+| 1   | Arm frequency capped |
+| 2   | Currently throttled |
+| 3   | Soft temperature limit active |
+| 16  | Under-voltage has occurred |
+| 17  | Arm frequency capped has occurred |
+| 18  | Throttling has occurred |
+| 19  | Soft temperature limit has occurred |
+
+#### measure_temp
+
+Returns the temperature of the SoC as measured by the on board temperature sensor
+
+#### measure_clock [clock]
+
+This returns the current frequency of the specified clock. The options are:
+
+| clock | Description |
+|:-----:|-------------|
+| arm   | ARM cores |
+| core  | VC4 scaler cores |
+| H264  | H264 block |
+| isp   | Image system Pipeline |
+| v3d   | 3D block |
+| uart  | UART |
+| pwm   | PWM block (analogue audio output) | 
+| emmc  | SD card interface |
+| pixel | Pixel valve |
+| vec | Analogue video encoder |
+| hdmi | HDMI |
+| dpi | Display Peripheral Interface |
```

Comment:
Is this list of clock-options the same for Pi3 and Pi4?

Reply:
Not sure....